.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Fixes #
+
+## Proposed Changes
```

Comment:
"Proposed" would be more appropriate for an issue, right? I usually go with a "Description" section because there may not be an issue associated, and a "Work Done" section listing the major items making up the pull request.